packages/workflow-core/src/lib/utils/types.ts

```diff
@@ -0,0 +1,8 @@
+import { JQTransformer } from './context-transformers/qj-transformer';
+import { JsonSchemaValidator } from './context-validator/json-schema-validator';
+import { AnyRecord } from '@ballerine/common';
+
+export type TContext = AnyRecord;
+
+export type TTransformers = JQTransformer;
```

Comment:
not in use

Reply:
this is what you should be using for transformation and validation